packages/ui/src/date-picker/DatePickerBase.tsx

```diff
@@ -1,11 +1,13 @@
-import { Paper, Popover, Typography } from '@material-ui/core';
+import { Grid, List, ListItem, Paper, Popover, Typography } from '@material-ui/core';
 import React, { useState } from 'react';
 import DayPicker, {
   CaptionElementProps,
   DayPickerProps,
   WeekdayElementProps,
 } from 'react-day-picker';
 
+export type Value = Date | [Date?, Date?];
```

Comment:
is it value used anywhere else? it's better to rename it to something more explicit like `DatePickerBaseValue` or do not export it